tests/integration/porter/conftest.py

```diff
@@ -0,0 +1,73 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+from base64 import b64decode
+
+import pytest
+
+from nucypher.crypto.umbral_adapter import PublicKey
+
+
+@pytest.fixture(scope='session')
+def random_federated_treasure_map_data():
+    random_bob_encrypting_key = PublicKey.from_bytes(
+        bytes.fromhex("026d1f4ce5b2474e0dae499d6737a8d987ed3c9ab1a55e00f57ad2d8e81fe9e9ac"))
+    random_treasure_map_id = "cc1c29dd2305483cc838cbcc5ecb5ef6edfd69ecadb0aa52b6b084b630989187"  # federated is 32 bytes
+    random_treasure_map = b64decode("VE0AAZ3jsXPynMD9dm+Fjwi49bxkOzUjwsNI0Y0p8bGB9F60OXGmJqibK0Ki4FSWti2Y"
```

Comment:
Would be helpful to have some note about how this treasure map was generated, for the event when its format changes. Or, if it's possible, maybe even generate it on the fly? It's a session-scoped fixture, after all.

Reply:
Added a note about the values being obtained from running the heartbeat demo locally. It is specific to porter tests, and generating it on the fly seems excessive.